test/controllers/viewCompanyInformation.controller.test.ts

```diff
@@ -138,6 +164,106 @@ describe('ViewCompanyInformationController', () => {
       assert.isTrue(htmlAssertHelper.selectorDoesNotExist('#submit'))
       assert.isTrue(htmlAssertHelper.selectorExists('#cannot-close-error'))
     })
+
+    it('should display error message when company is not of closable type', async () => {
```

Comment:
by moving the impl out to a service you will find that the unit tests become much simpler. Controller simply has to assert what happens when an error does / doesn't get returned. Service performs the individual checks without needing supertest, inversify or html checks